gatsby-theme-auth0/README.md

```diff
@@ -68,6 +68,24 @@ export default () => {
 };
 ```
 
+### AuthService methods
```

Comment:
Let's make this an `H2` instead of an `H3` and move it below the "Theme options" section